ruleset/rules/0915-win-powershell_rules.xml

```diff
@@ -19,20 +19,40 @@
     <rule id="91803" level="14">
-        <if_sid>91602</if_sid>
+        <if_sid>91802</if_sid>
         <field name="win.system.message" type="pcre2">CopyFromScreen</field>
         <options>no_full_log</options>
         <description>Screen capture method invoked from PowerShell script</description>
         <mitre>
             <id>T1113</id>
         </mitre>
     </rule>
+
+    <!-- Sample: {"win":{"eventdata":{"messageNumber":"1","messageTotal":"1","scriptBlockText":"Get-ADComputer -Identity 'cfo' -Properties *","scriptBlockId":"f6ea30ec-ff1e-4749-b7d8-0c2003a0c250"},"system":{"eventID":"4104","keywords":"0x0","providerGuid":"{A0C1853B-5C40-4B15-8766-3CF1C58F985A}","level":"5","channel":"Microsoft-Windows-PowerShell/Operational","opcode":"15","message":"\"Creating Scriptblock text (1 of 1):\r\nGet-ADComputer -Identity 'cfo' -Properties *\r\n\r\nScriptBlock ID: f6ea30ec-ff1e-4749-b7d8-0c2003a0c250\r\nPath: \"","version":"1","systemTime":"2021-06-30T22:29:22.450624400Z","eventRecordID":"5347","threadID":"920","computer":"bankdc.ExchangeTest.com","task":"2","processID":"6320","severityValue":"VERBOSE","providerName":"Microsoft-Windows-PowerShell"}}} -->
+    <rule id="91805" level="3">
+        <if_sid>91802</if_sid>
+        <field name="win.eventdata.scriptBlockText" type="pcre2">(?i)Get-ADComputer</field>
+        <description>Executed Powershell script "Get-ADComputer" executed</description>
+        <mitre>
+            <id>T1018</id>
+        </mitre>
+    </rule>
+
+    <!-- Sample: {"win":{"eventdata":{"messageNumber":"1","messageTotal":"1","scriptBlockText":"Get-NetUser","scriptBlockId":"b8633c48-c1ea-47b5-bad6-7766ff636abb"},"system":{"eventID":"4104","keywords":"0x0","providerGuid":"{A0C1853B-5C40-4B15-8766-3CF1C58F985A}","level":"5","channel":"Microsoft-Windows-PowerShell/Operational","opcode":"15","message":"\"Creating Scriptblock text (1 of 1):\r\nGet-NetUser\r\n\r\nScriptBlock ID: b8633c48-c1ea-47b5-bad6-7766ff636abb\r\nPath: \"","version":"1","systemTime":"2021-06-30T22:20:57.104893900Z","eventRecordID":"5329","threadID":"9440","computer":"bankdc.ExchangeTest.com","task":"2","processID":"3940","severityValue":"VERBOSE","providerName":"Microsoft-Windows-PowerShell"}}} -->
+    <rule id="91806" level="3">
+        <if_sid>91802</if_sid>
+        <field name="win.eventdata.scriptBlockText" type="pcre2">(?i)Get-NetUser</field>
```

Comment:
Why don't use osregex? By default it's case-insensitive

Reply:
I think the team agreed on suing pcre2